src/Types.hs

```diff
@@ -102,6 +104,13 @@ instance (SOP.Generic a, SOP.Generic b, SOP.Generic c) => SOP.Generic (Either3 a
 app2 :: (c -> d) -> (a -> b -> c) -> a -> b -> d
 app2 f g x y = f $ g x y
 
+sortOn :: Ord b => Getter a b -> [a] -> [a]
+sortOn l = sortBy (compare `on` view l)
+
+sortOnDesc :: Ord b => Getter a b -> [a] -> [a]
+sortOnDesc l = reverse . sortOn l
```

Comment:
> Please indicate the fact for the reverse order.

what do you mean?  the fact that the order is reversed is indicated by the `Desc` in the function name.